Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
> defined by various firmware projects as well as Skeinforge

Oh yes, we should not forget their immense contribution, bridging the worlds of CNC and RepRap.